src/app/settings/two-factor-recovery.component.ts

```diff
@@ -30,8 +30,8 @@ export class TwoFactorRecoveryComponent {
             '<code style="font-family: Menlo, Monaco, Consolas, \'Courier New\', monospace;">' +
             this.code + '</code></div>' +
             '<p style="text-align: center;">' + new Date() + '</p>');
+        w.onafterprint = w.close;
```

Comment:
Awesome, thanks! Can you please change this to it's own enclosure, `() => w.close();` to ensure context is maintained?

Reply:
@cscharf thank you for your feedback